angular-workspace/components/modal/modal.stories.ts

```diff
@@ -48,7 +45,29 @@ storiesOfModal(
       confirmFooter,
       loadingBody: loadingBody(templates).template,
       datePickerBody: datePickerBody(templates).template,
-    }),
+    };
+  },
+  decorator: (story) => {
```

Comment:
De decorator is voor het Angular component niet nodig, die maakt gebruik van het Web Component.

Reply:
Aangepast